user/api.go

```diff
@@ -828,7 +961,7 @@ func (a *Api) RefreshSession(res http.ResponseWriter, req *http.Request) {
 
 	if td.IsServer == false && td.DurationSecs > two_hours_in_secs {
```

Comment:
This if do not seems useful anymore.
like the ugly var name `two_hours_in_secs`

Reply:
Ok  I will push a fix on that.